stader-cli/node/update-operator-reward-address.go

```diff
@@ -81,11 +86,64 @@ func updateOperatorRewardAddress(c *cli.Context, operatorRewardAddress common.Ad
+	promptSuccessChangedRewardAndNextStep(cfg)
 
 	return nil
 }
 
-func promptNote() {
-	fmt.Printf("%sFollow these steps for your Reward address change: \nStep1: Propose the Reward address change by connecting your wallet \nto the Smart Contract using your EXISTING Reward address and the ProposeRewardAddress function. \nStep2: Confirm the Reward address change by connecting your wallet \nto the Smart Contract using your NEW Reward address and the ConfirmRewardAddressChange function. \n\n%sNote: Stader will continue to send reward to your existing Reward \naddress until you confirm the change using your new Reward Address%s", colorGreen, colorRed, colorReset)
+func promptHowToChangeReward(cfg *config.StaderConfig) {
+	network := cfg.StaderNode.Network.Value.(cfTypes.Network)
+	switch network {
+	case cfTypes.Network_Mainnet:
+		msg := `
+For node security, only your existing Reward Address can propose a change. To propose and confirm a Reward Address change, please use the PermissionlessNodeRegistry Smart Contract: https://etherscan.io/address/0xaf42d795A6D279e9DCc19DC0eE1cE3ecd4ecf5dD#writeProxyContract
+Follow these steps for your Reward Address change:
+Step 1: Propose the Reward Address change by connecting your Existing Reward Address wallet with the Smart Contract and execute the "ProposeRewardAddress" function.
+Step 2: Confirm the Reward Address change by connecting your New Reward Address wallet with the Smart Contract and execute the "ConfirmRewardAddressChange" function
+Please refer to the Reward Address change guide here - https://staderlabs.notion.site/staderlabs/Stader-ETHx-Reward-address-change-flow-Mainnet-Permissioned-3f2f83405cee47ca8d158a3409b90b31
+Note: Stader will continue to send rewards to your existing Reward Address until you confirm the change using your New Reward Address.`
+		fmt.Printf("%s %s %s\n", colorLightBlue, msg, colorReset)
+	case cfTypes.Network_Prater:
+		msg := `
+For node security, only your existing Reward Address can propose a change. To propose and confirm a Reward Address change, please use the PermissionlessNodeRegistry Smart Contract: https://goerli.etherscan.io/address/0xA8BbaD2C6d3F2a28EdB85Fc1c87B300EAC77A00d#writeProxyContract
+Follow these steps for your Reward Address change:
+Step 1: Propose the Reward Address change by connecting your Existing Reward Address wallet with the Smart Contract and execute the "ProposeRewardAddress" function.
+Step 2: Confirm the Reward Address change by connecting your New Reward Address wallet with the Smart Contract and execute the "ConfirmRewardAddressChange" function
+Please refer to the Reward Address change guide here - https://staderlabs.notion.site/staderlabs/Stader-ETHx-Reward-address-change-flow-Testnet-Permissioned-e8418ad8ac7f41ccae107f2996e85a96
+Note: Stader will continue to send rewards to your existing Reward Address until you confirm the change using your New Reward Address.`
```

Comment:
It would have been easier just to parameterize the smart contract url rather then the entire text

Reply:
Also put more specific links like: https://etherscan.io/address/0xaf42d795A6D279e9DCc19DC0eE1cE3ecd4ecf5dD#writeProxyContract#F4